source/blog/2019-01-07-ember-3-7.md

```diff
@@ -0,0 +1,100 @@
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.7
+Ember.js 3.7 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations. There is zero (0) new feature, zero (0) deprecations, and three (3) bugfixes in this 3.7 version.
+
+Note that in Ember.js 3.7 support for Node 4 has been explicitly dropped. This means that if you are on Node 4 and want to upgrade you will need to upgrade your version of Node first.
+
+#### New Features (0)
+
+There are no new features in Ember.js 3.7.
+
+
+#### Deprecations (0)
+
+Deprecations are added to Ember.js when an API will be removed at a later date. Each deprecation has an entry in the deprecation guide describing the migration path to a more stable API. Deprecated public APIs are not removed until a major release of the framework.
+
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon if you would like to upgrade your application without immediately addressing deprecations.
+
+For more details on changes in Ember.js 3.7, please review the [Ember.js 3.7.0 release page](https://github.com/emberjs/ember.js/releases/tag/v3.7.0).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 3.7
+
+No changes in Ember Data 3.7
+
+#### New Features (0)
+
+No new features introduced in Ember Data 3.7.
+
+#### Deprecations (0)
+
+No new deprecations introduced in Ember Data 3.7.
+
+
+For more details on changes in Ember Data 3.7, please review the
+[Ember Data 3.7.0 release page](https://github.com/emberjs/data/releases/tag/v3.7.0).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js applications.
+
+### Upgrading Ember CLI
+
+You may upgrade Ember CLI easily using the ember-cli-update project:
+
+```bash
+npm install -g ember-cli-update
+ember-cli-update
+```
+
+This utility will help you to update your app or add-on to the latest Ember CLI 3.7sion. You will probably encounter merge conflicts, in which the default behavior is to let you resolve conflicts on your own. For more information on the `ember-cli-update` project, see [the github README](https://github.com/ember-cli/ember-cli-update).
+
+While it is recommended to keep Ember CLI 3.7sions in sync with Ember and Ember Data, this is not required. After updating ember-cli, you can keep your current 3.7sion(s) of Ember or Ember Data by editing `package.json` to re3.7t the changes to the lines containing `ember-source` and `ember-data`.
+
+### Changes in Ember CLI 3.7
+
+#### New Features (0)
+
+No new features introduced in Ember CLI 3.7.
```

Comment:
I guess https://github.com/ember-cli/ember-cli/pull/8297 and https://github.com/ember-cli/ember-cli/pull/8286 are somewhat noteworthy